OpenTrons/opentrons-api
# feat(shared-data, app): add configureNozzleLayout command type in run log

# Overview
This PR adds support for the `configureNozzleLayout` command in run log


# Test Plan
1. Upload a partial-tip pickup protocol (you can find some here: https://github.com/Opentrons/test-protocols/tree/main/abr_testing/96_channel_partial_tip_protocols)
2. Start setup for the protocol and see that there will be an item in Run Log for the `configureNozzleLayout` command and that pickup tip commands now print a range of wells dependent on how many nozzles are being used at that time



# Changelog
1. Add `configureNozzleLayout` command type and supporting types 
2. Add a string for this command in `CommandText`
3. Update `pickUpTip` command text to use new util `getWellRange` that will return the range of wells that a pipette is going to pick up tips from based on the number of nozzles the pipette is currently using



# Review requests
Look through util and rest of code for sanity and test this out if you can!


# Risk assessment
Low
